### PR TITLE
Rc grade sheet endpoints

### PR DESCRIPTION
**Pull request by robert-czwartosz**
Hey,
I've done grade sheet endpoints and added them to main container 
so you can test how them works with command npm run start dev:
GET /grade/sheets/:id
GET /grade/sheets
POST /grade/sheets
PATCH /grade/sheets/reviewer/:id
PATCH /grade/sheets/reviewers/:id
PATCH /grade/sheets/mentor/grade/:id
PATCH /grade/sheets/reviewer/grade/:id
DELETE /grade/sheets/:id
Thanks in advance for any suggestions and bugs reports ;)

### CODE REVIEW

File: Api/Src/Controllers/GradeSheetController.ts [hunk 1]
```diff
@@ -0,0 +1,83 @@
+import GradeSheetModel from '../Models/GradeSheet';
+import GradeSheetService from '../Services/GradeSheetService';
+import { Request, Response } from 'express';
+import * as mongoose from 'mongoose';
+
+export default class UserController {
```
Comment (kjreczek) [inline Api/Src/Controllers/GradeSheetController.ts:6]: It's a default export, so doesn't cause any problems, but should be GradeSheetController

File: Api/Src/Controllers/GradeSheetController.ts [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+        const id = new mongoose.Types.ObjectId(req.params.id);
+        const mentorId = new mongoose.Types.ObjectId(req.body.mentorId);
+        const sheet = await this.gradeSheetService.addMentorReviewer(id, mentorId);
+        if(sheet === null) return res.status(404).json({message: 'Grade sheet not found'});
+        res.status(200).json({message: 'Mentor reviewer added'});
+    }
+
+    setMentorReviewers = async (req: Request, res: Response) => {
+        const id = new mongoose.Types.ObjectId(req.params.id);
+        if (req.body.mentorIds instanceof Array)
```
Comment (kjreczek) [inline Api/Src/Controllers/GradeSheetController.ts:34]: Missing negation here

File: Api/Src/Models/GradeSheet.ts [hunk 3]
```diff
@@ -0,0 +1,49 @@
+import * as mongoose from 'mongoose';
+
+export interface GradeSheet {
+    projectID: mongoose.Types.ObjectId;
+    angagement: number;
```
Comment (kjreczek) [inline Api/Src/Models/GradeSheet.ts:5]: engagement

File: Api/Src/Controllers/GradeSheetController.ts (hunk 1 above)
Comment (robert-czwartosz) [reply Api/Src/Controllers/GradeSheetController.ts]: Corrected 👍

File: Api/Src/Controllers/GradeSheetController.ts (hunk 2 above)
Comment (robert-czwartosz) [reply Api/Src/Controllers/GradeSheetController.ts]: Thanks 👍

File: Api/Src/Models/GradeSheet.ts (hunk 3 above)
Comment (robert-czwartosz) [reply Api/Src/Models/GradeSheet.ts]: Corrected 👍

**Review by kjreczek — APPROVED** [review]

**Review by JustynaSmoliga — COMMENTED** [review]
Looks good, you will resolve conflicts and I can merge pull request.

**Review by JustynaSmoliga — APPROVED** [review]

**Comment by JustynaSmoliga** [comment]
Unfortunately there are still some conflicts, but a moment ago everything was good, you should resolve conflicts again.

**Comment by robert-czwartosz** [comment]
Resolved

**Pull request merged**